packages/main/src/themes/StepInput.css

```diff
@@ -20,12 +20,65 @@
 	text-align: inherit;
 }
 
-:host(:not([value-state]):not([readonly]):not([disabled]):hover),
-:host([value-state=None]:not([readonly]):not([disabled]):hover) {
-    background-color: var(--sapField_Hover_Background);
-    border: 1px solid var(--sapField_Hover_BorderColor);
+:host(:not([value-state]):not([readonly]):not([disabled])){
+	box-shadow: none;
 }
```

Comment:
Even, with the fix in the input for the background image when disabled, it does not come out of the box below the buttons, you should adjust it

Reply:
done